sbysrc/sby_status.py

```diff
@@ -365,3 +369,11 @@ def combine_statuses(statuses):
         statuses.discard("UNKNOWN")
 
     return ",".join(sorted(statuses))
+
+def remove_db(path):
+    path = Path(path)
+    lock_exts = [".sqlite-wal", ".sqlite-shm"]
```

Comment:
These files always exist and never get deleted for me, so this always fails...

Reply:
From the [sqlite documentation](https://www.sqlite.org/tempfiles.html)
> The WAL file is created when the first connection to the database is opened and is normally removed when the last connection to the database closes. However, if the last connection does not shutdown cleanly, the WAL file will remain in the filesystem and will be automatically cleaned up the next time the database is opened.

And the SHM file should have the same lifetime.  Is it possible that you have something else keeping the database open?

Reply:
Can you test with the latest commit?